src/make.js

```diff
@@ -0,0 +1,13 @@
+module.exports.make = (function f() {
+  let params = [];
+  return function calc(...args) {
+    if (typeof args[0] === 'function') {
+      if (params.length === 0) return undefined;
+      const res = params.reduce(args[0]);
+      params = [];
+      return res;
+    }
+    params = params.concat([...args]);
```

Comment:
**params = params.concat([...args]);** the same as **params.push(...args)** in this case.